packages/api/ecs/async-operation/package.json

```diff
@@ -22,10 +22,10 @@
     "coverage": "python ../../../../scripts/coverage_handler/coverage.py"
   },
   "dependencies": {
-    "@cumulus/aws-client": "18.2.0",
-    "@cumulus/db": "18.2.0",
-    "@cumulus/es-client": "18.2.0",
-    "@cumulus/logger": "18.2.0",
+    "@cumulus/aws-client": "18.3.0",
```

Comment:
We got lucky here in that the dependency tree for this Docker image were all published as part of the incomplete 18.3.0 release. For the sake of getting things unblocked it made sense to just bump these dependencies to 18.3.0 instead of trying to publish some sort of alpha or version specific to this PR.

When 18.3.1 is released we should revisit this Dockerfile and bump the dependencies. 18.3.1 will not include aws-sdk v3 in the Lambda package so we want to verify everything still works - but it should.

Reply:
So temporarily bump to 18.3, merge to main, do the 18.3.1 release and bump these? That makes sense to me. Just want to be sure to catch it during the release as I don't know if npm update will 🤔. Either way, sounds like a plan.

Reply:
Yup that was my plan. It's the same thing we'd have to do if we pushed an "alpha" version it just so happened we got lucky and could use what was published of 18.3.0.